rpm/resalloc.spec.tpl

```diff
@@ -106,6 +106,20 @@ Requires(pre): /usr/sbin/useradd
 The %name-server package provides the resalloc server, and
 some tooling for resalloc administrators.
 
+%if %{with python3}
+%package webui
+Summary:    %sum - webui part
+
+Requires:   %default_python-%srcname = %version-%release
+Requires: %name-server
+Requires: python3-flask
```

Comment:
Can you please conditionalize this sub-package?  I'm sure python3-flask is not installable on RHEL7 at least.